### DISCUSSION TITLE
blog/minecraft-to-goldsrc

### DISCUSSION THREAD

**Discussion by giscus** [NotNite/giscus#5] [Announcements]
# blog/minecraft-to-goldsrc

https://notnite.com/blog/minecraft-to-goldsrc

**Comment by ZuidVolt** [comment]
This is a really cool project, great job 👍

**Comment by fredforce151** [comment]
Duuude I wanted this kind of stuff for almost 10 years now, hope you get this project finished and published asap I would love to create a ton of TFC maps created with Minecraft someday.

**Comment by keetsta** [comment]
That’s so cool
